### PR TITLE
fix: add missing `config` static method

### PR DESCRIPTION
**Pull request by JoaoPedroAS51**

### CODE REVIEW

File: src/StaticModel.js [hunk 1]
```diff
@@ -11,6 +11,13 @@ export default class StaticModel {
     return new this
   }
 
+  static config(config) {
+    let self = this.instance()
+    self.config(config)
+
+    return self
+  }
+
```
Comment (robsontenorio) [inline src/StaticModel.js:20]: @JoaoPedroAS51 can you elaborate about it?

What is it use case?

Comment (JoaoPedroAS51) [reply src/StaticModel.js:20]: This method was already added. It's in docs, but I forgot to add a static version of the method. See https://github.com/robsontenorio/vue-api-query/blob/dev/src/Model.js#L40

Comment (robsontenorio) [reply src/StaticModel.js:20]: Got i!

**Review by robsontenorio — APPROVED** [review]
@JoaoPedroAS51  approved.

**Pull request merged**